src/modules/secrets/Elsa.Secrets/Persistence/Decorators/InitializingSecretsStore.cs

```diff
@@ -0,0 +1,73 @@
+using Elsa.Persistence.Specifications;
+using Elsa.Secrets.Models;
+using Elsa.Services;
+using System.Threading.Tasks;
+using System.Threading;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Elsa.Secrets.Persistence.Decorators
+{
+    public class InitializingSecretsStore : ISecretsStore
```

Comment:
This looks like a blind copy & paste. There's no obvious reason to have this decorator class, unless I'm missing something.